src/main/java/io/specto/hoverfly/junit/core/Hoverfly.java

```diff
@@ -287,12 +294,19 @@ public SslConfigurer getSslConfigurer() {
         return sslConfigurer;
     }
 
+    public void verify(RequestMatcherBuilder requestMatcher, VerificationCriteria criteria) {
```

Comment:
You should overload this method to default to `times(1)` also, as people not using the rule would want it.